packages/@ember/-internals/glimmer/tests/integration/components/append-test.js

```diff
@@ -286,355 +278,6 @@ class AbstractAppendTest extends RenderingTestCase {
     );
   }
 
-  [`${testUnless(
```

Comment:
@NullVoxPopuli can you confirm that these tests were not erroneously marked with the deprecation?  At a glance it seems like they were coincidentally using the deprecated behavior but I'm not sure.